mxcube3/ui/components/SampleView/SampleControls.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
```

Comment:
Is this changing a rule for this file only ?

I think it's better if we follow the rules without exceptions.